### DISCUSSION TITLE
How to add a file in the dmg installation interface?

### DISCUSSION THREAD

**Discussion by huifer** [tauri-apps/tauri#11885] [General]
I want to add a shell file to the dmg installation interface to allow users to install brew dependencies. Or install brew dependencies during installation

**Comment by FabianLars** [comment]
I think this is what the `--add-file` option in `create-dmg` (the dmg tooling) is for. This has to be exposed in Tauri first before you can use it, so please open a feature request (a github issue) if you'd like to see this in tauri :)
